src/constants/url.constants.ts

```diff
@@ -0,0 +1,30 @@
+export const deriv = "deriv.com";
+export const derivMe = "deriv.me";
+export const derivBe = "deriv.be";
```

Comment:
isn't airbnb javascript standard suggesting that exported immutable constants should be UPPER_CASED?

Reply:
Currently, we aren't using this standard in the Utils. If I modify this file, I'd need to make the same changes across all constants. Let's talk about this later and then we can create a card to implement and enforce this standard across all our files.